### PR TITLE
feat: card grids

### PR DESCRIPTION
**Pull request by Valastiri**
## Summary
Implements the new card grid feature into various pages of our documentation. 

- Additional streamlining of pages.
- Custom on hover CSS for card grids
- At request of @Sleinmaster transitioned the documentation project page into it's own directory with two pages splitting the documentation.
  - Development Environment
  - Writing Documentation

### Location
- docs/fbw-a32nx/index.md
- docs/fbw-a32nx/support/reported-issues.md
- docs/dev-corner/development-projects

### Screenshots

Before
image

After
image

Hover State
image

New Directory
image

Discord username (if different from GitHub):
 valastiri

### CODE REVIEW

**Comment by Valastiri** [comment]
I think this is solid for now to get started and we can use this wherever applicable in the future.

**Comment by Valastiri** [comment]
Actually apologies, I should write up a how 2 guide in the development corner on the use case scenario and how to implement it. Will re-draft until that's complete.

**Comment by Sleinmaster** [comment]
> Actually apologies, I should write up a how 2 guide in the development corner on the use case scenario and how to implement it. Will re-draft until that's complete.

Maybe you can split the documentation project page while doing so. I think currently it's getting too large. 
Something like the Dev env setup should be on one page, the style and instructions like these on a second.

**Comment by Valastiri** [comment]
Great idea ill incorporate that during the edits

File: docs/dev-corner/development-projects/documentation-project/writing-documentation.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+    ```md title="Search Exclusion Meta Snippet"
+    ---
+    search:
+        exclude: true
+    ---
+    ```
+#### Card Grids
+
+Card grids is a recent feature added to mkdocs-material. This allows for a grid of cards to be displayed on a page. This is useful for displaying a list of links to other pages, or for displaying a list of links to external resources.
```
Comment (Sleinmaster) [inline docs/dev-corner/development-projects/documentation-project/writing-documentation.md]:
```suggestion
Card grids are a recent feature added to mkdocs-material. This allows for a grid of cards to be displayed on a page. This is useful for displaying a list of links to other pages, or for displaying a list of links to external resources. 
```

**Review by Sleinmaster — COMMENTED** [review]
That's the only thing I would change, otherwise looks sound.

**Pull request merged**